nototools/unittests/font_tests.py

```diff
@@ -592,9 +592,9 @@ def setUp(self):
             if name in self.whitelist:
                 continue
             master_a[name].draw(pen_a)
-            area_a = pen_a.unload()
+            area_a = pen_a.value
```

Comment:
Based on https://github.com/behdad/fonttools/pull/627#issuecomment-228894659 we need to reset the pen value after it's used here (and also change the calls to unload in shape_diff).